docker/docker-compose.yaml

```
@@ -3,50 +3,53 @@ services:
     image: ghcr.io/autowarefoundation/autoware:universe-localization-mapping
     container_name: autoware-map
     init: true
+    ipc: host
     environment:
-      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID:-0}
+      - ROS_DOMAIN_ID=${ROS_DOMAIN_ID}
     volumes:
-      - ${MAP_PATH:-$HOME/autoware_map}:/autoware_map:ro
+      - ${MAP_PATH}:/autoware_map:ro
     networks:
       - autoware-net
     command: >
       ros2 launch autoware_launch tier4_map_component.launch.xml
-      lanelet2_map_file:=${LANELET2_MAP_FILE:-lanelet2_map.osm}
+      lanelet2_map_file:=${LANELET2_MAP_FILE}
       map_path:=/autoware_map
-      pointcloud_map_file:=${POINTCLOUD_MAP_FILE:-pointcloud_map.pcd}
-      use_sim_time:=${USE_SIM_TIME:-true}
-      vehicle_model:=${VEHICLE_MODEL:-sample_vehicle}
-    restart: unless-stopped
+      pointcloud_map_file:=${POINTCLOUD_MAP_FILE}
```

Add localization pointcloud container arguments after merging https://github.com/autowarefoundation/autoware_launch/pull/1529 
```suggestion
      pointcloud_map_file:=${POINTCLOUD_MAP_FILE}
      launch_pointcloud_container:=${LAUNCH_POINTCLOUD_CONTAINER_LOCALIZATION}
      localization_pointcloud_container_name:=${POINTCLOUD_CONTAINER_NAME_LOCALIZATION}
```